python_ta/__init__.py

```diff
@@ -148,7 +146,24 @@ def _check(
                     file_linted=file_py,
                     load_default_config=load_default_config,
                 )
-                linter.set_reporter(current_reporter)
+
+                if not is_any_file_checked:
+                    current_reporter = linter.reporter
```

Comment:
Let me know if updating `current_reporter` is okay. My reasoning was that since we set up this linter using the same config file as the one on line 117, this is the same reporter, except it is guaranteed that the instances of `Message` from the config file errors have been converted to `NewMessage` instances.

Reply:
@sushimon yes this is great!